File: Python/SQLAlchemy_ORM/dal/entities/category.py
from sqlalchemy import Column, Integer, String, Text, LargeBinary, VARBINARY
from sqlalchemy.orm import relationship
from dal.data.base import Base

class Category(Base):
    __tablename__ = 'Categories'

    CategoryID = Column(Integer, primary_key=True, autoincrement=True)
    CategoryName = Column(String(15), nullable=False)
    Description = Column(Text)
    Picture = Column(String)

    products = relationship('Product', back_populates='category')

File: Go/GORM_ORM/dal/entities/category.go
package entities

type Category struct {
	CategoryID   int       `gorm:"primaryKey;autoIncrement;column:CategoryID"`
	CategoryName string    `gorm:"size:15;index:CategoryName;column:CategoryName"`
	Description  *string   `gorm:"type:ntext;column:Description"`
	Picture      []byte    `gorm:"type:image;column:Picture"`
	Products     []Product `gorm:"foreignKey:CategoryID"`
}
